less/home-page.less

```diff
@@ -1,9 +1,13 @@
 // Home Content
-
 .home {
 
```

Comment:
Since you have your navbar fixed, it would benefit the element after it, in this case, the `.home` container, to have a top margin as high as the height of the navbar. This would ensure the top of the home container does not hide behind the fixed navigation